optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -79,6 +81,12 @@
             Mel features extracted from the raw speech waveform. `(batch_size, feature_size, encoder_sequence_length)`.
 """
 
+VISION_ENCODER_INPUTS_DOCSTRING = r"""
+    Args:
+        pixel_values (`torch.FloatTensor`):
+            Featuers extracted from an Image. `(batch_size, num_channels, height, width)`.
```

Comment:
```suggestion
            Features extracted from an image. This tensor should be of shape `(batch_size, num_channels, height, width)`.
```

It's just RGB images no? Not "feature" in the sense of any preprocessing has been applied yet?

Reply:
I think the normalization and resize would be applied to the image

Reply:
right!